meson.build

```diff
@@ -1,6 +1,7 @@
 project('gnofract4d', 'cpp',
     license : 'BSD',
     default_options : ['cpp_std=c++17',
+        'cpp_args=-Wall',
```

Comment:
This should be done automatically via warning_level=3 - which you can change at any time with `meson configure --warnlevel n` where n is 0-3 inclusive. Please note that, for practicality reasons, once you create a configuration (that is what the build dir is), Meson will not update that configuration's settings based on the project defaults changing as they are considered only defaults for new configurations.

warning_level=3 (--warnlevel=3) implies: -Wall -Wextra -Wpedantic and a couple of other useful noisier warning flags

Reply:
Thanks. Removed cpp_args, also optimization which is controlled by buildtype apparently.

(Personally most of the time I build from a Gentoo ebuild, so it is all new each time. But still trying to preserve the run from the same directory as well as the develop and test from a local repository...)